### PR TITLE
Improving setup_maps.sh

### PR DESCRIPTION
**Pull request by src4026**
I think what I have done is correct, isn't it?

### CODE REVIEW

**Comment by LoneWolfHT** [comment]
Try what I just pushed, I'm on Windows atm

**Comment by src4026** [comment]
image

**Comment by LoneWolfHT** [comment]
Pushed a second change you can try

File: scripts/setup_maps.sh [hunk 1]
```diff
@@ -1,7 +1,10 @@
 #!/bin/bash
 set -e
 
-cd ..
+if [ basename "$PWD" -eq "scripts" ]; then
+	cd ..
+ fi
```
Comment (farooqkz) [inline scripts/setup_maps.sh]:
```suggestion
fi
```

**Comment by src4026** [comment]
I tried the one LV and it works now. Can I resolve the review you created @farooqkz ?

**Comment by src4026** [comment]
> Pushing a second change you can try

Works :+1:

**Comment by src4026** [comment]
Oh, cool. So the lines that you added work for whatever directory the `setup_maps.sh` script is run?

**Comment by LoneWolfHT** [comment]
> Oh, cool. So the lines that you added work in whatever directory the `setup_maps.sh` script is run?

In the main directory or in the scripts folder. I normally run git from the main directory so I always do `./scripts/setup_maps.sh`

**Comment by LoneWolfHT** [comment]
Weird, my Github didn't see either change go through until now

**Pull request merged**